Makefile

```diff
@@ -692,6 +692,30 @@ deploy/db:
 	@time timeout --foreground 3m bash -c "until $(OC) get pods -n $(NAMESPACE) | grep kas-fleet-manager-db | grep -v deploy | grep -q Running; do echo 'database is not ready yet'; sleep 10; done"
 .PHONY: deploy/db
 
+# Touch all the necessary files for fleet manager to start
+.PHONY: setup/secrets/empty
+setup/secrets/empty:
+	touch secrets/ocm-service.clientId
+	touch secrets/ocm-service.clientSecret
```

Comment:
I think it would be better to specify the `ocm-service.token` file here instead since this is the one that's mostly used for dev purposes.